### DISCUSSION TITLE
What does "meta" property contains?

### DISCUSSION THREAD

**Discussion by Mr-M1M3** [pocketbase/pocketbase#3280] [Q&A]
if authorized by oauth2, an extra property `meta` is included with the returned data. What does it contain?

**Comment by ganigeorgiev** [comment]
I'm not entirely sure that I understand what you are asking.

In the specific OAuth2 context, it contains fields related to your OAuth2 user (some standardized fields like  id, name, username, access and refresh tokens, etc. and also the raw user response).
